Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -96,16 +95,28 @@ function Main() {
 		LogMsg "$?: Installed required packages, $req_pkg"
 
 		# Start kernel compilation
-		LogMsg "Clone and compile new kernel from $hb_url to /usr/src/linux"
-		git clone $hb_url /usr/src/linux
-		LogMsg "$?: Cloned the kernel source repo in /usr/src/linux"
+		LogMsg "Cloning a new kernel from $hb_url to $linux_path"
+		git clone $hb_url $linux_path
```

Comment:
Might be a good idea to clone with a `--depth 1` argument as you don't care about old commits

Reply:
It's a good idea. I did not know of that option, but I changed to a stable branch after this during the test run-time. I am not sure how far it goes down. I will use that method in another test.